### PR TITLE
Add new function category 'Graph'

### PR DESCRIPTION
**Pull request by LinneaAndersson**

### CODE REVIEW

File: functions/src/main/java/org/neo4j/doc/FunctionDescriptionsGenerator.java [hunk 1]
```diff
@@ -60,7 +60,8 @@
         Temporal_instant_types( "Values of the <<cypher-temporal, temporal types>> -- _Date_, _Time_, _LocalTime_, _DateTime_, and _LocalDateTime_ -- can be created manipulated using the following functions:" ),
         Temporal_duration( "Duration values of the <<cypher-temporal, temporal types>> can be created manipulated using the following functions:" ),
         Spatial( "These functions are used to specify 2D or 3D points in a geographic or cartesian Coordinate Reference System and to calculate the geodesic distance between two points." ),
-        LOAD_CSV( "LOAD CSV functions can be used to get information about the file that is processed by `LOAD CSV`." );
+        LOAD_CSV( "LOAD CSV functions can be used to get information about the file that is processed by `LOAD CSV`." ),
+        Graph( "These functions returns information about the given graph." );
```
Comment (LinneaAndersson) [inline functions/src/main/java/org/neo4j/doc/FunctionDescriptionsGenerator.java]: @tobias-johansson Do you have a better suggestion of description 😄

File: functions/src/main/java/org/neo4j/doc/FunctionDescriptionsGenerator.java [hunk 2]
```diff
@@ -60,7 +60,8 @@
         Temporal_instant_types( "Values of the <<cypher-temporal, temporal types>> -- _Date_, _Time_, _LocalTime_, _DateTime_, and _LocalDateTime_ -- can be created manipulated using the following functions:" ),
         Temporal_duration( "Duration values of the <<cypher-temporal, temporal types>> can be created manipulated using the following functions:" ),
         Spatial( "These functions are used to specify 2D or 3D points in a geographic or cartesian Coordinate Reference System and to calculate the geodesic distance between two points." ),
-        LOAD_CSV( "LOAD CSV functions can be used to get information about the file that is processed by `LOAD CSV`." );
+        LOAD_CSV( "LOAD CSV functions can be used to get information about the file that is processed by `LOAD CSV`." ),
```
Comment (AlexicaWright) [inline functions/src/main/java/org/neo4j/doc/FunctionDescriptionsGenerator.java]:
```suggestion
        LOAD_CSV( "LOAD CSV functions can be used to get information about the file processed by `LOAD CSV`." ),
```

**Review by AlexicaWright — COMMENTED** [review]
Small suggestion :)

**Review by AlexicaWright — APPROVED** [review]
👍  LGTM now!

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @LinneaAndersson & @AlexicaWright 

👍

**Pull request merged**